client/components/orderable-list/index.js

```diff
@@ -0,0 +1,43 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React from 'react';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+
+const ListItem = ( { children, className } ) => {
+	return (
+		<li className={ classNames( 'orderable-list__item', className ) }>
+			<div className="orderable-list__drag-handle" />
+			{ children }
+		</li>
+	);
+};
+
+const OrderableList = ( { className, children } ) => {
+	const childrenArray = React.Children.toArray( children );
+	const showDragHandles = 1 < childrenArray.length;
+
+	return (
+		<ul
+			className={ classNames(
+				'orderable-list',
+				{ 'show-drag-handles': showDragHandles },
```

Comment:
I noticed other classe names with booleans using a `has`/`is` prefix - what are your thoughts on renaming the class `has-drag-handles` or `is-orderable` or `is-sortable`?